src/controllers/breaches.js

```diff
@@ -1,10 +1,11 @@
 import { mainLayout } from '../views/layouts/main.js'
 import { breaches } from '../views/partials/breaches.js'
-import { setBreachResolution, setBreachesResolved, updateBreachStats } from '../db/tables/subscribers.js'
+import { setBreachResolution, updateBreachStats } from '../db/tables/subscribers.js'
 import { getUserEmails } from '../db/tables/email_addresses.js'
 import { getBreachesForEmail, filterBreaches } from '../utils/hibp.js'
 import { filterBreachDataTypes } from '../utils/breach-resolution.js'
 import { getSha1 } from '../utils/fxa.js'
+
 async function breachesPage (req, res) {
   // TODO: remove: to test out getBreaches call with JSON returns
   const breachesData = await getAllEmailsAndBreaches(req.user, req.app.locals.breaches)
```

Comment:
Should `breachesData` now instead await `getBreaches()` below?

Reply:
No, because `getBreaches` is used for the API (returns a callback of JSON). What we want here is the underlying function. I'll have to refactor this function `getAllEmailsAndBreaches` and name it something else during the Q1 refactor